uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -94,29 +96,41 @@ sub lookupNextVisitLabel {
     }
     return $visitLabel;
 }
-                 
+
 ################################################################
 ##################### extract_tarchive #########################
 ################################################################
 =pod
-Most important function now. Gets the tarchive and 
+Most important function now. Gets the tarchive and
 extracts it so data can actually be uploaded
 =cut
 sub extract_tarchive {
     my $this = shift;
-    my ($tarchive) = @_;
-    print "Extracting tarchive $tarchive in $this->{TmpDir} \n" 
-    if $this->{verbose};
+    my ($tarchive, $tarchive_id, $verbose) = @_;
```

Comment:
theoretically, I should not need to pass on $verbose here (given it can be obtained from this->{verbose}), but it only worked if I did what I did...
in 2 more places down as well...